src/db/spanner/models.rs

```diff
@@ -783,6 +796,157 @@ impl SpannerDb {
         }
     }
 
+    pub async fn get_quota_usage_async(
+        &self,
+        params: params::GetQuotaUsage,
+    ) -> Result<results::GetQuotaUsage> {
+        let check_sql = if self.quota_enabled {
+            "SELECT IFNULL(total_bytes,0), IFNULL(count,0)
+            FROM user_collections
+           WHERE fxa_uid = @fxa_uid
+             AND fxa_kid = @fxa_kid
+             AND collection_id = @collection_id"
+        } else {
```

Comment:
I think this clause is dead code at this point? We only call it when quota is enabled, even in the proposed (commented out) test. If so it could just `assert!(self.quota_enabled)`